packages/storage/package.json

```diff
@@ -48,6 +48,7 @@
     "cids": "^1.1.4",
     "crypto-browserify": "^3.12.0",
     "event-emitter": "^0.3.5",
+    "moment": "^2.29.1",
```

Comment:
Is it possible we used a lighter library light [`dayjs`](https://day.js.org/). I feel moment js is quite large and might make the sdk bundle become larger than expected. Especially since we are only using the format feature of moment at the moment.

Reply:
Fixed in [9f008c9](https://github.com/FleekHQ/space-sdk/pull/19/commits/9f008c9a641c36671ec1c3113569bf4188102210).